part3.md

```diff
@@ -0,0 +1,80 @@
+## Part 3 - Hello Kotlin
+
+ * Make sure that you have Kotlin JS installed (in `functions` sub-folder)
+```
+npm install -g kotlin
+```
+
+ * Open IntelliJ IDEA and create new pure Kotlin project in root folder. Use Kotlin(JavaScript) and Gradle in wizard. As a result you shold have `functions` folder as a subfolder of you project. 
+
+ * Add following block or `build.gradle`:
+``` groovy
+compileKotlin2Js.kotlinOptions {
+    moduleKind = "commonjs"
+    outputFile = "functions/build/index.js"
+}
+```
+
+ * Write same `testPush` function in `src/main/kotlin/Index.kt` in basic KotlinJS:
```

Comment:
Now write the function `testPush` in KotlinJS. Create a file in `src/main/kotlin/Index.kt`